host/xtest/regression_4000_data.h

```diff
@@ -8742,4 +8742,79 @@ static const uint8_t ed25519ph_rfc_8032_7_3_out[] = {
 	0xaa, 0xd1, 0x1c, 0x2a, 0x26, 0x08, 0x34, 0x06
 };
 
+/* RSA AES wrapped key
+ * Test data generated as follows:
+ * - generate target AES-256 key (target.aes)
+ * - generate wrapping RSA kaypair (we use ac_rsassa_vect2 in this case)
```

Comment:
s/kaypair/keypair/

Reply:
fixed in [a4f3f71](https://github.com/OP-TEE/optee_test/pull/627/commits/a4f3f7118946a5f2be78e74c1bc82c65445f9752)